README.md

```diff
@@ -2,91 +2,94 @@
 
 ## Key generation
 
-### Add the ssh key to your DO account
+1. Add the [ssh key](https://cloud.digitalocean.com/account/security)
+   to your DO account
 
-[Digital ocean account security](https://cloud.digitalocean.com/account/security)
+2. You'll need to generate an access token in DigitalOcean's control panel
 
->You find a guide on the website DO
+   [Account api token](https://cloud.digitalocean.com/settings/applications)
 
-### You'll need to generate an access token in DigitalOcean's control panel
+3. After add digitalocean API token in file
 
-[Account api token](https://cloud.digitalocean.com/settings/applications)
-
-### After add digitalocean API token in file
-
-```bash
-~/.do/access_token
-```
+    ```bash
+      ~/.do/access_token
+    ```
```

Comment:
I think good note that `this token should be private and not shared with anyone in any cases`